config/_default/menus/main.en.yaml

```diff
@@ -7883,41 +7883,46 @@ menu:
       identifier: product_analytics
       parent: digital_experience_heading
       weight: 50000
-    - name: User Retention
-      url: product_analytics/user_retention
+    - name: Vizualizing with Charts
+      url: product_analytics/charts
       parent: product_analytics
-      identifier: pa_user_retention
+      identifier: pa_charts
       weight: 1
-    - name: Journeys
-      url: product_analytics/journeys
-      parent: product_analytics
-      identifier: pa_journeys
-      weight: 2
     - name: Pathway Diagrams
```

Comment:
- Most other instances of this term are plural "Pathways", so maybe this should be "Pathways Diagrams"?
- When you go to this page, the title is just "Pathways", so maybe update that to match what's in the side nav. (Titles and side nav don't always have to match exactly, but there doesn't seem to be a reason why they shouldn't here.)